change/@fluentui-react-checkbox-28af053d-dd0e-4df2-815d-5a8ade1eeaa4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31640&commitId=a1f3df36477b8f29ea1e62f705a1d2729ea2ad28&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|
|Avatar Converged.size+image+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Added|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>